reddit_service_activity/counter.py

```diff
@@ -29,9 +29,7 @@ def _current_slice():
     return int(time.time() // _SLICE_LENGTH)
```

Comment:
The docstring at the top of this file is very helpful!  I should have read it before googling around for HLL stuff.